doc/source/releases/20.05_announce_user.rst

```diff
@@ -0,0 +1,105 @@
+
+===========================================================
+May 2020 Galaxy Release (v 20.05)
+===========================================================
+
+.. include:: _header.rst
+
+Highlights
+===========================================================
+
+**Mass-Spec datatypes**
+  @bernt-matthias, can you write a bit about this?
+
+**New Interactive tools**
+  @astrovsky01, can you write this ?
+
+**Feature3**
```

Comment:
I'd recommend the following highlights, maybe in this order:

1. Tool recommendations (even if not on every galaxy, can be, and it is a user requested/facing feature.)
2. New ITs
3. Fixed the rule builder scrolling (huge bug for many medium power users.), todo find xref.
3. OIDC Login "yay social auth, login with fav provider blah blah"
  - https://github.com/galaxyproject/galaxy/pull/9060
  - Connect/disconnect interface
  - https://github.com/galaxyproject/galaxy/pull/9162 logout
  - elixir aai saw improvements


ITs is good, mass-spec is maybe too specific for this announcement?

I'd recommend against startup speed + tool testing improvements, these are really end user announcements and should focus on things people who don't run or develop galaxy, people who just show up to .org/.eu

Reply:
Fair, I'll move / drop those.